components/ide/code/codehelper/main.go

```diff
@@ -282,6 +282,9 @@ func prepareServerMain(wsInfo *supervisor.WorkspaceInfoResponse) error {
 	}
 	domain := url.Hostname()
 	b = bytes.ReplaceAll(b, []byte("https://*.vscode-cdn.net"), []byte(fmt.Sprintf("https://%s https://*.%s", domain, domain)))
+	// disable CSP to align with regular workspace, otherwise it blocks custom schemes, like vscode:, vscode-insiders: or jetbrains-gateway:
+	b = bytes.ReplaceAll(b, []byte("Content-Security-Policy"), []byte("Disabled___Content____Security____Policy"))
```

Comment:
maybe disable/comment it on gp-code/main rathrr than doing here, we don't use it anyway

Reply:
will do